integration-test/src/test/java/org/sagebionetworks/IT049FileHandleTest.java

```diff
@@ -75,8 +84,15 @@
 	private File imageFile;
 	private Project project;
 
+	// Hard-coded dev test bucket
+	private String googleCloudBucket = "dev.test.gcp-storage.sagebase.org";

```

Comment:
Best/correct way to do this? Used spring in an autowired test but seems to not be the pattern in integration tests